### DISCUSSION TITLE
Firefox android tidak mendukung VPN dan DNS aplikasi

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1845] [Q&A]
Saya kok menemukan keganjilan ya pada Firefox android 
Mengapa tidak support aplikasi VPN dan aplikasi DNS seperti cloudflare dan quad9
VPN akan mati sendiri setelah sekitar kurang lebih 10menit
Saya menggunakan pengaman antivirus Kaspersky 
APAKAH ADA YG SAMA DENGAN SAYA??

**Comment by 3xploiton3** [comment]
tujuan menggnakan 2 aplikasi vpn dan aplikasi dns apa?

kebanyakan aplikasi vpn sudah support custom dns encrypted, 

antivirus di android ? saya rasa tidak perlu, jarang sekali mendengar virus android, kecuali spyware pegasus yg pakai 0 day
